docs/concepts/how-to-grant-api-access.md

```diff
@@ -0,0 +1,183 @@
+
+# Granting Users Access to API Resources in OLM
+
+When operators or cluster extensions are installed through OLM, they often provide Custom Resource Definitions (CRDs) that expose new API resources. By default, cluster administrators may have full access to manage these resources, but non-admin users often require specific permissions to create, view, or edit these custom resource objects.
```

Comment:
```suggestion
When operators or cluster extensions are managed via OLM, they often provide Custom Resource Definitions (CRDs) that expose new API resources. Typically, cluster administrators hold full management access to these resources by default, whereas non-administrative users may lack sufficient permissions. Such users often need designated permissions to create, view, or edit these Custom Resources.
```

Maybe?

Reply:
yeah, reads better and "managed" is more correct. thanks!